liquidctl/driver/asus_ryuo.py

```diff
@@ -0,0 +1,64 @@
+"""liquidctl driver for the ASUS Ryuo liquid coolers.
+
+Copyright Bloodhundur and contributors
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+
+import logging
+from typing import List
+
+from liquidctl.driver.usb import UsbHidDriver
+from liquidctl.error import ExpectationNotMet
+from liquidctl.util import clamp, rpadlist
+
+_LOGGER = logging.getLogger(__name__)
+
+_REPORT_LENGTH = 65
+_PREFIX = 0xEC
+_CMD_SET_FAN_SPEED = 0x2A
+
+_REQUEST_GET_FIRMWARE = (0x82, 0x02)
+_STATUS_FIRMWARE = "Firmware version"
+
+
+class AsusRyuoTest(UsbHidDriver):
+    """Driver for ASUS ROG Ryuo I 240 (fan control only)."""
+
+    _MATCHES = [
+        (0x0B05, 0x1887, "ASUS ROG Ryuo I 240", {}),
+    ]
+
+    feature_set_fixed_speed = True
+
+    def initialize(self, **kwargs):
+        _LOGGER.info("No initialization required")
+
+    def get_status(self, **kwargs):
+        msg = self._request(*_REQUEST_GET_FIRMWARE)
+        fw_string = bytes(msg[2:]).split(b'\x00')[0].decode('ascii', errors='ignore')
+        return [
+            (_STATUS_FIRMWARE, fw_string, ''),
+        ]
```

Comment:
Since the firmware version doesn't change often, it's better to return it from `initialize` instead of `get_status`.